src/main/java/org/folio/rest/impl/AbstractHelper.java

```diff
@@ -95,6 +106,54 @@ public void closeHttpClient() {
     return processingErrors.getErrors();
   }
 
+  protected boolean isCheckin(PoLine poLine) {
+    return defaultIfNull(poLine.getCheckinItems(), false);
+  }
+
+  public CompletableFuture<String> updatePoLineReceiptStatus(PoLine poLine, ReceiptStatus status, HttpClientInterface httpClient,
```

Comment:
This method can be moved to helper utilities class

Reply:
Done